View/LoginScreen/login_screen.py

```diff
@@ -11,3 +11,5 @@ def model_is_changed(self) -> None:
         The view in this method tracks these changes and updates the UI
         according to these changes.
         """
+    def store_input(self): 
```

Comment:
i guess dapat pareho kayo ni rachell ng method name diyan since same ng ginagawa which is kunin yung text sa textfield